lib/airbrake-ruby/sync_sender.rb

```diff
@@ -33,18 +21,15 @@ def initialize(config)
     # @param [Airbrake::Notice] endpoint
     # @return [Hash{String=>String}] the parsed HTTP response
     def send(notice, endpoint = @config.endpoint)
-      response = nil
       req = build_post_request(endpoint, notice)
       https = build_https(endpoint)
 
-      begin
-        response = https.request(req)
-      rescue *HTTP_ERRORS => ex
-        @config.logger.error("#{LOG_LABEL} HTTP error: #{ex}")
-        return
-      end
-
+      response = https.request(req)
       Response.parse(response, @config.logger)
+
+    rescue => ex
```

Comment:
But if all the async workers die due to a temporal error condition, we are left with no workers and then we fall back to synchronous behaviour when we don't expect it (like we were saying in https://github.com/airbrake/airbrake-ruby/pull/50)

What's the harm in capturing all exceptions in the function to avoid workers dying unnecessary and falling back to synchronous behaviour?